.github/workflows/bench-pr.yml

```diff
@@ -26,9 +26,8 @@ jobs:
     needs: label_trigger
     runs-on:
       - runs-on=${{ github.run_id }}
-      - family=c7i.8xlarge
+      - family=c6id.8xlarge
```

Comment:
Might be good to add a comment that the disk size is now implied by the instance type.